stp_zmq/zstack.py

```diff
@@ -446,7 +446,7 @@ def _verifyAndAppend(self, msg, ident):
         except (UnicodeDecodeError, InvalidMessageExceedingSizeException) as ex:
             errstr = 'Message will be discarded due to {}'.format(ex)
             frm = self.remotesByKeys[ident].name if ident in self.remotesByKeys else ident
-            logger.error("Got from {} {}".format(frm, errstr))
+            logger.warning("Got from {} {}".format(frm, errstr))
```

Comment:
Please also have a look at `batched.py`, `signAndSerialize`. There is debug log there which should be warning or error.